codebuild/samples/pkcs11-pubsub-linux.sh

```diff
@@ -24,14 +24,14 @@ openssl pkcs8 -topk8 -in /tmp/privatekey.pem -out /tmp/privatekey.p8.pem -nocryp
 softhsm2-util --import /tmp/privatekey.p8.pem --token my-token --label my-key --id BEEFCAFE --pin 0000
 
 # build and run sample
-pushd $CODEBUILD_SRC_DIR/samples/mqtt/pkcs11_pub_sub
+pushd $CODEBUILD_SRC_DIR/samples/mqtt/pkcs11_connect
```

Comment:
Maybe these scripts should be renamed now that we're saying "connect" instead of "pubsub"